crates/core/migrations/2022-08-15-172917_add_compound_key_to_candymachine_creators/up.sql

```diff
@@ -0,0 +1,4 @@
+-- Your SQL goes here
+alter table candy_machine_creators
+drop constraint candy_machine_creators_pkey,
+add constraint candy_machine_creators_compound_pkey primary key (candy_machine_address, creator_address);
```

Comment:
hmm, do we care that this can fail when migrating backwards?

Reply:
you should just add `select false` in down.sql because the on_conflict code need to be changed as well if the migration is reverted